02/PrintRecord.cpp

```diff
@@ -0,0 +1,24 @@
+#include <iostream>
+#include <sstream>
+#include <string>
+#include "PrintRecord.h"
+
+using namespace std;
+
+namespace samples
+{
+	void PrintRecordExample()
+	{
+		string record = "Pope Kim A12345678 18";
+
+		istringstream inputStream(record);
+		ostringstream outputStream;
+
+		for (size_t i = 0; i < 4; i++)
```

Comment:
inputstream >> firstName >> lastName >> studentID >> score;

then do outputstream << with nice formatting :)  i dno't wanna just simply reprint it.